local-up-cluster.sh broken by enhance kubeproxy init flag commit

```
/home/dims/go/src/k8s.io/kubernetes/_output/local/bin/linux/amd64/hyperkube flag redefined: config
panic: /home/dims/go/src/k8s.io/kubernetes/_output/local/bin/linux/amd64/hyperkube flag redefined: config

goroutine 1 [running]:
k8s.io/kubernetes/vendor/github.com/spf13/pflag.(*FlagSet).AddFlag(0xc4200b6000, 0xc420bc8dc0)
	/home/dims/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/github.com/spf13/pflag/flag.go:817 +0x9d2
k8s.io/kubernetes/vendor/github.com/spf13/pflag.(*FlagSet).VarPF(0xc4200b6000, 0xaf96b60, 0xc420c4ebb0, 0x4bf8e67, 0x6, 0x0, 0x0, 0x4c7f9b5, 0x23, 0x74a62f)
	/home/dims/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/github.com/spf13/pflag/flag.go:800 +0x120
k8s.io/kubernetes/vendor/github.com/spf13/pflag.(*FlagSet).VarP(0xc4200b6000, 0xaf96b60, 0xc420c4ebb0, 0x4bf8e67, 0x6, 0x0, 0x0, 0x4c7f9b5, 0x23)
	/home/dims/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/github.com/spf13/pflag/flag.go:806 +0x8e
k8s.io/kubernetes/vendor/github.com/spf13/pflag.(*FlagSet).StringVar(0xc4200b6000, 0xc420c4ebb0, 0x4bf8e67, 0x6, 0x0, 0x0, 0x4c7f9b5, 0x23)
	/home/dims/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/github.com/spf13/pflag/string.go:37 +0xa3
k8s.io/kubernetes/cmd/kube-proxy/app.(*Options).AddFlags(0xc420c4ebb0, 0xc4200b6000)
	/home/dims/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/cmd/kube-proxy/app/server.go:122 +0x74
k8s.io/kubernetes/cmd/kube-proxy/app.NewProxyCommand(0x1)
	/home/dims/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/cmd/kube-proxy/app/server.go:357 +0x250
main.NewHyperKubeCommand.func3(0xc4201a9440)
	/home/dims/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/cmd/hyperkube/main.go:102 +0x26
main.commandFor(0x7ffc9f1b3dec, 0x9, 0xc42052d680, 0xc4202f4380, 0x7, 0x7, 0x8ba6520)
	/home/dims/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/cmd/hyperkube/main.go:71 +0xdf
main.main()
	/home/dims/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/cmd/hyperkube/main.go:63 +0x230
```

--- a/cmd/kube-proxy/BUILD
+++ b/cmd/kube-proxy/BUILD
@@ -22,6 +22,7 @@ go_library(
         "//cmd/kube-proxy/app:go_default_library",
         "//pkg/client/metrics/prometheus:go_default_library",
         "//pkg/version/prometheus:go_default_library",
+        "//vendor/github.com/spf13/pflag:go_default_library",
         "//vendor/k8s.io/apiserver/pkg/util/flag:go_default_library",
         "//vendor/k8s.io/apiserver/pkg/util/logs:go_default_library",
     ],

--- a/cmd/kube-proxy/app/server.go
+++ b/cmd/kube-proxy/app/server.go
@@ -354,7 +354,7 @@ with the apiserver API to configure the proxy.`,
 		glog.Fatalf("unable to create flag defaults: %v", err)
 	}
 
-	opts.AddFlags(pflag.CommandLine)
+	opts.AddFlags(cmd.Flags())
 
 	cmd.MarkFlagFilename("config", "yaml", "yml", "json")
 

--- a/cmd/kube-proxy/proxy.go
+++ b/cmd/kube-proxy/proxy.go
@@ -17,10 +17,13 @@ limitations under the License.
 package main
 
 import (
+	goflag "flag"
 	"fmt"
 	"math/rand"
 	"os"
 	"time"
+
+	"github.com/spf13/pflag"
 
 	utilflag "k8s.io/apiserver/pkg/util/flag"
 	"k8s.io/apiserver/pkg/util/logs"
@@ -34,7 +37,12 @@ func main() {
 
 	command := app.NewProxyCommand()
 
-	utilflag.InitFlags()
+	// TODO: once we switch everything over to Cobra commands, we can go back to calling
+	// utilflag.InitFlags() (by removing its pflag.Parse() call). For now, we have to set the
+	// normalize func and add the go flag set by hand.
+	pflag.CommandLine.SetNormalizeFunc(utilflag.WordSepNormalizeFunc)
+	pflag.CommandLine.AddGoFlagSet(goflag.CommandLine)
+	// utilflag.InitFlags()
 	logs.InitLogs()
 	defer logs.FlushLogs()
 

